src/app/filecoin-plus/allocators/components/NoAllocatorsComponents.tsx

```diff
@@ -0,0 +1,19 @@
+import { FIL_PLUS_URLS } from '@/constants/siteMetadata'
+
+import { Button } from '@/components/Button'
+
+export function NoAllocatorsComponent() {
+  return (
+    <div className="mx-auto space-y-6 text-center">
+      <h2 className="text-xl font-bold">Allocator Data Unavailable</h2>
+      <p className="max-w-readable">
+        We’re having trouble fetching the allocator data. For the latest
+        information, please check the Allocators list on the Filecoin Plus
+        Registry.
+      </p>
+      <Button href={FIL_PLUS_URLS.allocators.site} variant="primary">
+        Go To FIlecoin Plus Registry
```

Comment:
Use https://capitalizemytitle.com/ when needed. I uncapitalized `to` and there was a typo in `Filecoin`

```suggestion
        Go to Filecoin Plus Registry
```